Description: Fixes disutils (adds a setup.py, etc.).
Author: Ghe Rivero <ghe@debian.org>
Forwarded: no

--- /dev/null
+++ b/setup.py
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(name='python-novnc',
+      version='0.4+dfsg+1+20130425+git4973b9cc80',
+      description='NoVNC python libraries',
+      author='Ghe Rivero',
+      author_email='ghe.rivero@stackops.com',
+      url='http://www.stackops.com',
+      packages = ['novnc'],
+      package_dir = {'novnc':'utils'},
+      py_modules=['wsproxy','websocket','web','json2graph','img2js'],
+)
--- /dev/null
+++ b/utils/__init__.py
@@ -0,0 +1 @@
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
