Description: Changes call to websockify to use PATH
Author: Simon Bazley <sibaz@sibaz.com>
Forwarded: no

--- a/utils/launch.sh
+++ b/utils/launch.sh
@@ -28,6 +28,8 @@
 CERT=""
 WEB=""
 proxy_pid=""
+#WEBSOCKIFY=${HERE}/websockify
+WEBSOCKIFY=websockify
 
 die() {
     echo "$*"
@@ -102,7 +104,7 @@
 fi
 
 echo "Starting webserver and WebSockets proxy on port ${PORT}"
-${HERE}/websockify --web ${WEB} ${CERT:+--cert ${CERT}} ${PORT} ${VNC_DEST} &
+${WEBSOCKIFY} --web ${WEB} ${CERT:+--cert ${CERT}} ${PORT} ${VNC_DEST} &
 proxy_pid="$!"
 sleep 1
 if ! ps -p ${proxy_pid} >/dev/null; then
